formidling/src/main/java/no/nav/ung/sak/formidling/innhold/EndringHøySatsInnholdBygger.java

```diff
@@ -0,0 +1,56 @@
+package no.nav.ung.sak.formidling.innhold;
+
+import java.math.RoundingMode;
+import java.time.LocalDate;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import jakarta.enterprise.context.Dependent;
+import jakarta.inject.Inject;
+import no.nav.fpsak.tidsserie.LocalDateInterval;
+import no.nav.fpsak.tidsserie.LocalDateTimeline;
+import no.nav.ung.kodeverk.dokument.DokumentMalType;
+import no.nav.ung.sak.behandlingslager.behandling.Behandling;
+import no.nav.ung.sak.behandlingslager.ytelse.UngdomsytelseGrunnlagRepository;
+import no.nav.ung.sak.behandlingslager.ytelse.sats.Sats;
+import no.nav.ung.sak.formidling.template.TemplateType;
+import no.nav.ung.sak.formidling.template.dto.EndringHøySatsDto;
+import no.nav.ung.sak.formidling.vedtak.DetaljertResultat;
+
+@Dependent
+public class EndringHøySatsInnholdBygger implements VedtaksbrevInnholdBygger {
+
+    private UngdomsytelseGrunnlagRepository ungdomsytelseGrunnlagRepository;
+
+    private static final Logger LOG = LoggerFactory.getLogger(EndringHøySatsInnholdBygger.class);
+
+    @Inject
+    public EndringHøySatsInnholdBygger(
+            UngdomsytelseGrunnlagRepository ungdomsytelseGrunnlagRepository) {
+        this.ungdomsytelseGrunnlagRepository = ungdomsytelseGrunnlagRepository;
+    }
+
+    public EndringHøySatsInnholdBygger() {
+    }
+
+    @Override
+    public TemplateInnholdResultat bygg(Behandling behandling, LocalDateTimeline<DetaljertResultat> resultatTidslinje) {
+
+        LocalDate satsendringsdato = resultatTidslinje.getMinLocalDate();
```

Comment:
Kunne denne hatt en kommentar som sier hvorfor satsendringsdato er den første datoen i tidslinjen? 

Jeg antar at det er fordi resultattidslinjen er fra vilkårstidslinjene, og at vi setter fra-dato i vilkårstidslinjene for behandlingen til 25-årsdagen

Reply:
ja. Eni, det er litt magisk.